src/test/java/net/sf/jabref/event/TestHierarchicalEventListener.java

```diff
@@ -0,0 +1,36 @@
+package net.sf.jabref.event;
+
+import net.sf.jabref.model.entry.BibEntry;
+
+import com.google.common.eventbus.Subscribe;
+
+public class TestHierarchicalEventListener {
```

Comment:
I think, this is calls is only used in testing and does not test code in the main JabRef code. Just remove it.